src/lazy/binary/raw/v1_1/e_expression.rs

```diff
@@ -64,11 +38,14 @@ pub struct BinaryEExpression_1_1<'top> {
     cache: Option<&'top [ValueExpr<'top, BinaryEncoding_1_1>]>,
     macro_ref: MacroRef<'top>,
     bitmap_bits: u64,
-    // The index of `input` at which the bitmap can be found. If there is no bitmap, this index
-    // will be the beginning of the encoded arguments.
+    // This index is the first position after the opcode and address.
+    // If the e-expression has a length prefix, it will begin at this position in `input`.
+    length_offset: u8,
```

Comment:
🗺️ This type had 6 bytes of padding, so adding another `u8` field did not increase its size.